cis/com.b2international.snowowl.snomed.cis/src/com/b2international/snowowl/snomed/cis/memory/DefaultSnomedIdentifierService.java

```diff
@@ -61,7 +61,7 @@
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(DefaultSnomedIdentifierService.class);
 
-	private final Index store;
+	public final Index store;
```

Comment:
Fields to the internals of a class are intentionally marked with the private keyword and should not be accessed like this. This creates a false image and allows developers to manage IDs by directly accessing the internal store API. For a use case like this, I recommend the use of an SPI, an interface that allows access to the internals of a service and allows the developers to use those internals. A good name for this interface would be `InternalSnomedIdentiferService`.

Reply:
At the very least an accessor method should be added. The script command will not run with anything else than `DefaultSnomedIdentifierService` as the implementation at the moment.